Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -967,6 +967,17 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Verify if string matches the regex.
+    ///
+    /// - Parameter regex: Regex to verify.
+    /// - Returns: true if string matches the regex.
+    func matches(regex: NSRegularExpression) -> Bool {
+        let range = NSRange(location: 0, length: utf16.count)
+        return regex.firstMatch(in: self, options: [], range: range) != nil
```

Comment:
Could options be an extra optional parameter with a default `[]`? That would be useful?

Reply:
👍